coretex/coretex/run/run.py

```diff
@@ -399,41 +399,41 @@ def createQiimeArtifact(self, rootArtifactFolderName: str, qiimeArtifactPath: Pa
     @classmethod
     def run(
```

Comment:
let's rename this method to `start` this way we never call `Run.run`, but `Run.start`

Reply:
also rename `runLocal` to `startLocal`